src/fen_util.rs

```diff
@@ -144,6 +213,29 @@ pub fn get_fen_castles(chessboard: &Chessboard) -> String {
     if rights.is_empty() { "-".to_string() } else { rights }
 }
 
+/// Retrieves the en passant square from the chessboard and returns it in algebraic notation.
+///
+/// # Arguments
+///
+/// - `chessboard`: A reference to the `Chessboard` struct to get the en passant square from.
+///
+/// # Returns
+///
+/// A string representing the en passant square in algebraic notation ("-" if no en passant square).
+///
+/// # Example
+///
+/// ```
+/// use kn_o3::Chessboard;
+///
+/// let mut chessboard = Chessboard::new();
+/// chessboard.en_passant = 29; // Corresponding square for "e3"
+/// let en_passant_square = get_fen_passant(&chessboard);
+/// assert_eq!(en_passant_square, "e3");
+/// ```
+///
+/// The function retrieves the en passant square from the chessboard and returns it in algebraic notation.
+/// If there is no en passant square (0), it returns "-".
```

Comment:
self explanatory if you know what a fen is

Reply:
For this one you could literally say `-` if no en passant otherwise the coordinate.